turbo.json

```diff
@@ -0,0 +1,39 @@
+{
+  "pipeline": {
+    "codegen:entrypoints": {
+      "outputs": [
+        "links",
+        "shared",
+        "ssg",
+        "adapters",
+        "dist",
+        "http",
+        "observable",
+        "rpc",
+        "subscription"
+      ]
```

Comment:
maybe we can automate the process that when a new entrypoint is added, it's added to the outputs here so it can be cached 🤔